docker-compose.yml

```diff
@@ -83,7 +91,7 @@ services:
 
   sneakers:
     image: ruby:3.2.2
-    command: bash -c "cd /app && WORKERS=PolicyWorker rake sneakers:run"
+    command: bash -c "cd /app && WORKERS=PolicyWorker,PaymentWorker rake sneakers:run"
```

Comment:
Acho que aqui vc não precisa listar os workers, ele vai pegar magicamente todos os q estão na pasta workers

Reply:
Pior que o meu não estava funcionando antes de adicionar isso, eu segui o exemplo da app to thiago que também foi configurada assim, daí funcionou